druid/src/widget/click.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A clickable [`Controller`] widget.
+//!
+//! [`Controller`]: struct.Controller.html
+
+use crate::widget::Controller;
+use crate::{Data, Env, Event, EventCtx, LifeCycle, LifeCycleCtx, Widget};
+
+/// A clickable [`Controller`] widget. Pass this and a child widget to a
```

Comment:
I would maybe mention the relationship with button; that this is intended for places where you want something to be clickable, but don't want to use the standard button widget?